indico/modules/events/registration/models/forms.py

```diff
@@ -281,6 +291,16 @@ class RegistrationForm(db.Model):
         )
     )
 
+    # relationship backrefs:
+    # - in_event_acls (EventPrincipal.registration_form)
+
+    def __contains__(self, user):
+        return user is not None and any(user in registration.user for registration in self.registrations)
```

Comment:
`user == registration.user`

Reply:
registrations that are not in complete/unpaid state should not be taken into account